ui/src/timeMachine/utils/insertFunction.test.ts

```diff
@@ -18,10 +18,10 @@ describe('insertFunction', () => {
   test('formatFunctionForInsert', () => {
```

Comment:
these tests are okay. they pass. but they're kind of hard to read and they're kind of barebones. the power of rspec-like tests (which jest is) is that we can describe things in human language for people to read. `insertFunction generateImport` is kind of meh compared to something like [this](https://github.com/influxdata/influxdb/blob/master/ui/src/variables/utils/getWindowVars.test.ts) where it's much more natural to read: `getWindowPeriodVariable should return null when passed an empty query string`. 

writing sentences out like this also clarifies our thinking by forcing us to state in words what the code we wrote is doing.